drivers/ethernet/eth_stm32_hal.c

```diff
@@ -43,7 +48,6 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define PHY_BSR  ((uint16_t)0x0001U)  /*!< Transceiver Basic Status Register */
 #define PHY_LINKED_STATUS  ((uint16_t)0x0004U)  /*!< Valid link established */
 
-#define GET_FIRST_DMA_TX_DESC(heth)	(heth->Init.TxDesc)
```

Comment:
I would prefer not to delete this macro. But it does not makes the functionality worse. Thus please let me know the reason to delete it.

Reply:
The difficulty with that macro is that for H7 it becomes all this code, which essentially has two outputs (cur_tx_desc_idx, dma_tx_desc):

```
	uint32_t cur_tx_desc_idx;

	cur_tx_desc_idx = heth->TxDescList.CurTxDesc;
	dma_tx_desc = (ETH_DMADescTypeDef *)heth->TxDescList.TxDesc[cur_tx_desc_idx];
```

Do you have a preference of how it should be handled instead?

Reply:
Ok. Fine. Let Dekrete it. The Code is anyway correct.